### ISSUE TITLE
`consistent-boolean-name`: Vue ref/computed with boolean prefix incorrectly flagged

### ISSUE THREAD

**Issue by jesse-kaufman** [sindresorhus/eslint-plugin-unicorn#3553]
Vue refs and computed refs starting with boolean prefix are incorrectly flagged as needing to be boolean. I believe this started when I updated from 69.0.0 to 71.0.0 and I'm currently seeing it 71.1.0.

`consistent-boolean-name`

```js
// result: `isBranch` starts with `is`, so it should be boolean
const isBranch = ref(false) 

// result: `hasDepartment` starts with `has`, so it should be boolean
const hasDepartment = computed(() => departments.length > 0)
```
